docs/source/set_dev_env_with_vagrant.rst

```diff
@@ -34,4 +50,12 @@ environment to develop Nitrate,
 
 visit http://127.0.0.1:8087 with your favourite web browser.
 
+* Create a user through web-interface
+
+* In order to make it admin, hack MySQL query:
+  ::
+    mysql -u nitrate
+    USE nitrate
+    UPDATE auth_users SET is_staff=1,is_active=1,is_superuser=1 WHERE id=1
```

Comment:
A correct way to create a useruser is to use `createsuperuser`. Although you can do it in these steps, it should be avoided in practice.

Reply:
Is it written in the doc?

Reply:
@GodfatherThe  Please have a look at #264 . The Vagrant file and provision script are rewrite totally. All the steps to set up a development environment are automated in ansible playbook. `set_dev_env_with_vagrant.rst` is also updated. Can you take a look at if that updated version is clear enough for you?